_includes/tiles.html

```diff
@@ -0,0 +1,15 @@
+<section class="section">
+	<div class="tile_container">
+	{% for work in site.caseStudies %}
+	{% if work.published == "y" %}
+		<figure class="tile tile-honey">
+			<img src="{{work.thumbnail}}" alt="img01" class="tile_image"/>
```

Comment:
Can you change the alt tag on this please. Maybe `picture of {{ work.title }}` or something like that